exchanges/bybit/bybit_wrapper.go

```diff
@@ -417,120 +404,48 @@ func (by *Bybit) UpdateTradablePairs(ctx context.Context, forceUpdate bool) erro
 
 // UpdateTickers updates the ticker for all currency pairs of a given asset type
 func (by *Bybit) UpdateTickers(ctx context.Context, assetType asset.Item) error {
+	enabled, err := by.GetEnabledPairs(assetType)
+	if err != nil {
+		return err
+	}
+	format, err := by.GetPairFormat(assetType, false)
+	if err != nil {
+		return err
+	}
+	var ticks *TickerData
 	switch assetType {
-	case asset.Spot:
-		ticks, err := by.GetTickersV5(ctx, "spot", "", "")
+	case asset.Spot, asset.USDCMarginedFutures,
+		asset.USDTMarginedFutures, asset.CoinMarginedFutures, asset.Options:
+		var baseCoin string
+		if assetType == asset.Options {
+			baseCoin = "BTC"
```

Comment:
There are more options than just BTC:
https://api.bybit.com/v5/market/tickers?category=option&baseCoin=ETH

Reply:
Bybit only supports two types of options thankfully. So please update this to make two requests, one for BTC and one for ETH. You'll likely need to change this a little bit to make these two requests compared to all the other assets